src/main/java/banking/database/DBManager.java

```diff
@@ -0,0 +1,59 @@
+package banking.database;
+
+import org.sqlite.SQLiteDataSource;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DBManager {
+
+    public static final String SQL_CREATE_TABLE = """
+            CREATE TABLE IF NOT EXISTS account(
+                id INTEGER PRIMARY KEY,
+                cardNumber TEXT UNIQUE,
+                cardPIN TEXT,
+                balance INTEGER DEFAULT 0
+            );
+            """;
+
+    private final SQLiteDataSource dataSource;
+    private Connection connection;
+
+    public DBManager(String fileName) {
+        dataSource = new SQLiteDataSource();
+        dataSource.setUrl("jdbc:sqlite:" + fileName);
+        tryToCreateConnection();
+        executeQuery();
+    }
+
+    private void executeQuery() {
+        try (Statement statement = getConnection().createStatement()) {
+            statement.executeUpdate(SQL_CREATE_TABLE);
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void tryToCreateConnection() {
+        try {
+            this.connection = dataSource.getConnection();
+        } catch (SQLException e) {
+            System.err.println("Cannot create a connection with database!");
```

Comment:
And what’s next? After printing error?

Reply:
Edited. Now the program stops in that exceptional situations.I guess that's expected behavior.